### DISCUSSION TITLE
mistake

### DISCUSSION THREAD

**Discussion by ninjadoc107** [captcha-challenger/hcaptcha-model-factory#21] [Q&A]
when trying to run it returns this error, what can it be?

![image](https://user-images.githubusercontent.com/110185860/188666454-be8724d6-2bc8-44b2-a436-07eefced4f74.png)

**Comment by QIN2DIM** [comment]
```bash
pip install numpy==1.21.5 opencv-python==4.5.5.62

```

**Comment by ninjadoc107** [comment]
> ```shell
> opencv-python==4.5.5.62
> ```

error persists, I'm using python 3.10.5, is there a specific version I need to use?

**Comment by QIN2DIM** [comment]
`pip install opencv-python==4.6.0.66`

**Comment by ninjadoc107** [comment]
![image](https://user-images.githubusercontent.com/110185860/188785605-3fb7fa41-8de4-4509-af6e-d9b93b6037e8.png)

...

**Comment by yyyanbj** [comment]
What is your `torch` version? If you use a low version, you need to downgrade `numpy>=1.17,<1.21`.

**Comment by ninjadoc107** [comment]
> `pip install opencv-python==4.6.0.66`

I'm using this version, which version of numpy should I use?

**Comment by yyyanbj** [comment]
This has nothing to do with the `OpenCV` version, depending on your `PyTorch` version. You can find it by `pip show torch`.

**Comment by ninjadoc107** [comment]
> Isso não tem nada a ver com a `OpenCV`versão, dependendo da sua `PyTorch`versão. Você pode encontrá-lo por `pip show torch`.

version: 1.12.1

**Comment by yyyanbj** [comment]
Downgrade your torch version can solve all your problem. Suggest to use stable version `1.8.2`.

**Comment by ninjadoc107** [comment]
> Fazer o downgrade da sua versão da tocha pode resolver todos os seus problemas. Sugira usar a versão estável `1.8.2`.

and how can i download this version?

**Comment by QIN2DIM** [comment]
@ninjadoc107 screenshot, `pip show numpy`，`pip show opencv-python`

**Comment by yyyanbj** [comment]
https://pytorch.org/get-started/locally/

**Comment by ninjadoc107** [comment]
> @ninjadoc107captura de tela, `pip show numpy`，`pip show opencv-python`

![image](https://user-images.githubusercontent.com/110185860/188793490-5a5478e9-277c-4010-a370-c267be8d5192.png)

**Comment by QIN2DIM** [comment]
and `😂 pip show torch`

**Comment by ninjadoc107** [comment]
> and `😂 pip show torch`

![image](https://user-images.githubusercontent.com/110185860/188794043-ce38e311-efb6-46e8-b4e0-75fb3cd5af35.png)

**Comment by QIN2DIM** [comment]
It is recommended to create a pure runtime environment with a tool like conda.

**Comment by ninjadoc107** [comment]
> Recomenda-se criar um ambiente de tempo de execução puro com uma ferramenta como conda.
oh i give up lol

**Comment by QIN2DIM** [comment]
or try todo that:

#ps: python3.10

`pip install numpy==1.23.2 opencv-python==4.6.0.66`

**Comment by ninjadoc107** [comment]
> or try todo that:
> 
> `pip install numpy==1.23.2 opencv-python==4.6.0.66`

it was, bro thank you very much

**Comment by ninjadoc107** [comment]
> or try todo that:
> 
> `pip install numpy==1.23.2 opencv-python==4.6.0.66`

![image](https://user-images.githubusercontent.com/110185860/188795875-5ea93b32-53c8-4d4c-8d28-d7840466e9d4.png)


at the end of it all returns this, where is the image missing?

I did everything on the wiki

**Comment by QIN2DIM** [comment]
wiki? why not google or stackoverflow 😂

**Comment by ninjadoc107** [comment]
> wiki? por que não google ou stackoverflow😂

stop laughing at me kasgkagkaskg

**Comment by QIN2DIM** [comment]
There is a problem with your data set. You have files in other formats than images in your folder.

**Reply by ninjadoc107** [reply]
it worked here bro, thanks for your patience and this is not an issue like the room tag says, this is a conversation already, lol
